src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -153,7 +160,9 @@
         .define(CONSUMER_THREADS_CONFIG, Type.INT, CONSUMER_THREADS_DEFAULT,
                 Importance.MEDIUM, CONSUMER_THREADS_DOC)
         .define(CONSUMER_INSTANCE_TIMEOUT_MS_CONFIG, Type.INT, CONSUMER_INSTANCE_TIMEOUT_MS_DEFAULT,
-                Importance.LOW, CONSUMER_INSTANCE_TIMEOUT_MS_DOC);
+                Importance.LOW, CONSUMER_INSTANCE_TIMEOUT_MS_DOC)
+        .define(SIMPLE_CONSUMER_POOL_SIZE_CONFIG, Type.INT, SIMPLE_CONSUMER_POOL_SIZE_DEFAULT,
+                Importance.LOW, SIMPLE_CONSUMER_POOL_SIZE_DOC);
```

Comment:
Similar question on the importance level. If the default value doesn't cover 99% of workloads, it should probably be marked at least medium.